doc/release-notes-1.10.rst

```diff
@@ -92,15 +102,23 @@ Build and Infrastructure
   You *will* need to port your own out-of-tree scripts and Makefiles to
   CMake.
 
-Libraries
-*********
+Libraries / Subsystems
+***********************
+
+* Add random subsystem: Some "random" drivers are not drivers at all: they just
+  implement the function `sys_rand32_get()`.  Move those to a random subsystem
+  in preparation for a reorganization.
 
-* details ...
```

Comment:
Please note here that TinyCrypt has been updated to 0.2.8.  No notable changes, since most of them were already integrated in Zephyr during the 1.9 release.